src/screens/BallotEjectScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import Main, { MainChild } from '../components/Main'
+import Prose from '../components/Prose'
+import Button from '../components/Button'
+import MainNav from '../components/MainNav'
+
+const Columns = styled.div`
+  display: flex;
+  > div {
+    margin-right: 1em;
+    &:first-child {
+      flex: 1;
+    }
+    &:last-child {
+      margin-right: 0;
+    }
+    img {
+      max-width: 100%;
+      height: 83vh;
+    }
+  }
+`
+
+const RectoVerso = styled.div`
+  display: flex;
+  & > * {
+    &:first-child {
+      margin-right: 1em;
+    }
+  }
+  img {
+    max-width: 100%;
+    height: 83vh;
+  }
+`
+
+interface Props {
+  continueScanning: () => void
+}
+
+const BallotEjectScreen = ({ continueScanning }: Props) => {
+  return (
+    <React.Fragment>
+      <MainNav>
+        <Button small onPress={continueScanning}>
+          Continue Scanning Batch
+        </Button>
+      </MainNav>
+      <Main>
+        <MainChild maxWidth={false}>
+          <Columns>
+            <Prose maxWidth={false}>
+              <h1>Remove This Ballot</h1>
+              <p>Human review is required for this last scanned ballot.</p>
+              <p>
+                Once the ballot has been removed, press the button to continue
+                scanning.
+              </p>
+            </Prose>
+            <RectoVerso>
+              <Prose>
+                <h4>Front</h4>
+                <p>
+                  <img src="/eject/p1.jpg" alt="p1" />
```

Comment:
Pass these image URLs in?

Reply:
I was going to… but then wasn't sure what the data model would be… would the image URLs be part of some object?

Reply:
At this level I think just having the URLs be passed in is fine. Higher up, yeah, there's some data model.